assets/javascripts/test_result.js

```diff
@@ -586,6 +586,20 @@ function setCurrentPreviewFromStepLinkIfPossible(stepLink) {
   }
 }
 
+function githashToLink(value) {
+  var hashes_regex = /(\b[0-9a-f]{9}\b)/gms;
+  var githashes = value.match(hashes_regex);
+  var repo = 'https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/';
+  var stat_regex = /(\<a.+?)(?=(\<a))/gms;
+  if (githashes == null) {
```

Comment:
The `==` operator should be avoided most of the times, maybe just check for an array instead:
```suggestion
  if (!Array.isArray(githashes)) {
```

Reply:
this change is ready for review